pymc3/distributions/dist_math.py

```diff
@@ -67,6 +68,12 @@ def bound(logp, *conditions, **kwargs):
     -------
     logp with elements set to -inf where any condition is False
     """
+
+    # If called inside a model context, see if bounds check is disabled
+    model = modelcontext(kwargs.get("model"))
```

Comment:
Im more curious than opposed, but why not just skip the bound check where its called instead of no opping in the check bounds function?

Reply:
In that case we would need to add an `if`-clause to every Distribution.

Reply:
ugh yea that would suck. This is better